### PR TITLE
Update Sample App with New Design

### PR DESCRIPTION
**Pull request by demirsoya**
Notes about the changes: 

- Update sample app design
- Update app logo
- Fix xamarin compiling issues
- Update Android SDK to latest (7.0.9) (We have another PR to update SDKs to latest and check telemetry. However, Android SDK was needed to be updated here due to unplanned Androidx migration. Telemetry check and IOS SDK update will take place in the next PR)
- Migrate to Androidx
- Add missing libraries like Kotlin.Couroutines, Moshi etc...
- Update all packages (Xamarin.Forms is now 4.8.0.xxx)
- Android target sdk upgraded to 29 (it can be upgraded to 30 if we install it to mac mini)

Some screenshots showing the design and app changes.

Screenshot 2021-01-07 at 17 38 31

Screenshot 2020-11-18 at 14 25 42

Screenshot 2020-11-18 at 14 27 54

### CODE REVIEW

**Comment by hiteshjain29** [comment]
> Screenshot 2020-11-18 at 14 26 11
> Screenshot 2020-11-18 at 14 25 42
> Screenshot 2020-11-18 at 14 27 54
> 
> * Update sample app new with the new look provided by designers.

File: UsabillaBindings/Xamarin.Usabilla.Sample/MainPage.xaml [hunk 1]
```diff
@@ -1,48 +1,63 @@
...
             HorizontalOptions="FillAndExpand"
-            Clicked="OnLoadFormClicked" />
+            Margin="32"
+            Spacing="16"
+            VerticalOptions="Center"
+            RelativeLayout.WidthConstraint=
+            "{ConstraintExpression Type=RelativeToParent,
+                                Property=Width,
+                                Factor=1}"
+            RelativeLayout.HeightConstraint=
```
Comment (hiteshjain29) [inline UsabillaBindings/Xamarin.Usabilla.Sample/MainPage.xaml:22]: Screenshot 2020-12-21 at 17 16 13

@demirsoya, As per this wireframe, logo should be in the bottom and text will be on the top.

**Review by hiteshjain29 — CHANGES_REQUESTED** [review]

File: UsabillaBindings/Xamarin.Usabilla.Sample/MainPage.xaml (hunk 1 above)
Comment (hiteshjain29) [reply UsabillaBindings/Xamarin.Usabilla.Sample/MainPage.xaml:27]: @demirsoya i still see little bit of differences.

Comment (demirsoya) [reply UsabillaBindings/Xamarin.Usabilla.Sample/MainPage.xaml:27]: what exactly you wanted to be changed?

**Review by hiteshjain29 — APPROVED** [review]

**Pull request merged**